custom_components/frigate/binary_sensor.py

```diff
@@ -141,6 +150,87 @@ def icon(self) -> str:
         return get_dynamic_icon_from_type(self._obj_name, self._is_on)
 
 
+class FrigateAudioSensor(FrigateMQTTEntity, BinarySensorEntity):  # type: ignore[misc]
+    """Frigate Audio Sensor class."""
+
+    def __init__(
+        self,
+        config_entry: ConfigEntry,
+        frigate_config: dict[str, Any],
+        cam_name: str,
+        audio_name: str,
+    ) -> None:
+        """Construct a new FrigateAudioSensor."""
+        self._cam_name = cam_name
+        self._audio_name = audio_name
+        self._is_on = False
+        self._frigate_config = frigate_config
+
+        super().__init__(
+            config_entry,
+            frigate_config,
+            {
+                "state_topic": {
+                    "msg_callback": self._state_message_received,
+                    "qos": 0,
+                    "topic": (
+                        f"{self._frigate_config['mqtt']['topic_prefix']}"
+                        f"/{self._cam_name}/audio/{self._audio_name}"
+                    ),
+                },
+            },
+        )
+
+    @callback  # type: ignore[misc]
+    def _state_message_received(self, msg: ReceiveMessage) -> None:
+        """Handle a new received MQTT state message."""
+        self._is_on = msg.payload == "ON"
+        self.async_write_ha_state()
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID for this entity."""
+        return get_frigate_entity_unique_id(
+            self._config_entry.entry_id,
+            "audio_sensor",
+            f"{self._cam_name}_{self._audio_name}",
+        )
+
+    @property
+    def device_info(self) -> dict[str, Any]:
+        """Return device information."""
+        return {
+            "identifiers": {
+                get_frigate_device_identifier(self._config_entry, self._cam_name)
+            },
+            "via_device": get_frigate_device_identifier(self._config_entry),
+            "name": get_friendly_name(self._cam_name),
+            "model": self._get_model(),
+            "configuration_url": f"{self._config_entry.data.get(CONF_URL)}/cameras/{self._cam_name}",
+            "manufacturer": NAME,
+        }
+
+    @property
+    def name(self) -> str:
+        """Return the name of the sensor."""
+        return f"{self._audio_name} sound"
```

Comment:
Minor: If there are different kinds of audio (which I'm guessing there are), should we use big endian naming here for readability, e.g.:

front_door_sound_speech
front_door_sound_crying
front_door_sound_of_deathstar

What do you think? This doesn't really match what we do with labels & label counts (e.g. 'front_door_person_count' and not 'front_door_count_person'), but I think this makes some sense as 'sound' is a whole new class of detection.

Reply:
Yeah I wasn't thinking of it like count. I was thinking of it like how there is `person_occupancy`

Reply:
Yes, that's a much better counterexample. I think if we could go back in time, "front_door_occupancy_person" would have been better. Ah well, I'm fine with this for consistency.